assets/css/styles.css

```diff
@@ -0,0 +1,311 @@
+body {
+  margin: 0px;
+  background-color: #081c15;
+  font-family: "Dancing Script", cursive;
+  font-family: "Josefin Sans", sans-serif;
+  font-family: "Lato", sans-serif;
+  font-family: "Poppins", sans-serif;
+  font-family: "Roboto", sans-serif;
+  font-family: "Roboto Condensed", sans-serif;
+  font-family: "Smokum", cursive;
+  font-family: "Source Sans Pro", sans-serif;
+}
```

Comment:
There are a lot of font-family here, and they are all overwriting each other, so the one that is last overwrites the rest. Consider removing all the others if they aren't being used anymore.